bin/generate_tests.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3.7
+import argparse
+import json
+import logging
+import os
+import re
+import sys
+import tempfile
+from glob import glob
+from subprocess import check_call, CalledProcessError, DEVNULL
+
+from jinja2 import Environment, FileSystemLoader, TemplateNotFound
+
+VERSION = '0.1.0'
+
+DEFAULT_SPEC_LOCATION = os.path.join('..', 'problem-specifications')
+RGX_WORDS = re.compile(r'[-_\s]|(?=[A-Z])')
+
+logging.basicConfig()
+logger = logging.getLogger('generator')
+logger.setLevel(logging.WARN)
+
+
+def snake_case(string):
+    return '_'.join(RGX_WORDS.split(string)).lower()
+
+
+def title_case(string):
```

Comment:
It's a bit more accurate to call this `camel_case` rather than `title_case` ... title case usually means "Every Significant Word Gets an Uppercase First Letter but with Spaces Maintained".